File: ros2_ws/src/my_cpp_pkg/src/node_template.cpp
#include "rclcpp/rclcpp.hpp"

class MyCustomNode : public rclcpp::Node // Modify name
{
public:
    MyCustomNode() : Node("node_name") // Modify name
    {
        //
        RCLCPP_INFO(this->get_logger(), "My Custom Node has been started.");
    }

private:
};

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<MyCustomNode>(); // Modify name
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}

File: ros2_ws/src/my_py_pkg/my_py_pkg/node_template.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node


class MyCustomNode(Node):  # Modify name
    def __init__(self):
        super().__init__("node_name")  # Modify name
        self.get_logger().info("My Custom Node has been started.")


def main(args=None):
    rclpy.init(args=args)
    node = MyCustomNode()  # Modify name
    rclpy.spin(node)
    rclpy.shutdown()


if __name__ == "__main__":
    main()
